mmv1/third_party/terraform/services/firebase/resource_firebase_android_app_sweeper.go

```diff
@@ -14,7 +15,8 @@ import (
 )
 
 func init() {
-	resource.AddTestSweepers("FirebaseAndroidApp", &resource.Sweeper{
+	_, filename, _, _ := runtime.Caller(0)
+	resource.AddTestSweepers("FirebaseAndroidApp"+filename, &resource.Sweeper{
```

Comment:
Shall we wrap this into a function in another package? I expect this will need to be called for every sweeper as we migrate so it makes sense to create this distinction now.

Reply:
My plan is to first merge this PR in. If it does not break the tools, then I will wrap this into a function in a separate PR. Do you prefer to make it a separate PR now?